install_config/install/deploy_router.adoc

```diff
@@ -1113,74 +1113,87 @@ endif::[]
+Modify it as desired.
 
-Create a configmap:
-====
+. Create a configMap:
++
+[source,bash]
 ----
 $ oc create configmap customrouter --from-file=haproxy-config.template
 ----
-====
-The configMap contains a copy of your modified *_haproxy-config.template_* file.
-The router dc is next modified to mount the configMap as a file and the
-`TEMPLATE_FILE` environment variable is modified to point to it. This can be
-done using "oc env" and "oc volume" or alternaitvely by editing the router dc.
++
+The `customrouter` configMap now contains a copy of the modified
+*_haproxy-config.template_* file.
 
-====
+. Modify the router deployment configuration to mount the configMap
+as a file and point the `TEMPLATE_FILE` environment variable to it.
+This can be done via `oc env` and `oc volume` commands,
+or alternatively by editing the router dc.
++
+Using `oc` commands::
++
+[source,bash]
 ----
-$ oc env dc/router TEMPLATE_FILE=/var/lib/haproxy/conf/custom/haproxy-config.template
-$ oc volume dc/router --add --overwrite --name=config-volume \
-  --mount-path=/var/lib/haproxy/conf/custom \
-  --source='{"configMap": { "name": "customrouter"}}'
+$ oc env dc/router \
+    TEMPLATE_FILE=/var/lib/haproxy/conf/custom/haproxy-config.template
+$ oc volume dc/router --add --overwrite \
+    --name=config-volume \
+    --mount-path=/var/lib/haproxy/conf/custom \
+    --source='{"configMap": { "name": "customrouter"}}'
 ----
++
+Editing the router dc::
++
+Use `oc edit dc router` to edit the router deployment configuration
+with a text editor.
++
 ====
-
-Alternatively, edit the router dc to mention the configMap in all the needed places.
-In the `*spec.container.env*` section add the `TEMPLATE_FILE` environment variable to
-point to the mounted *_haproxy-config.template_*. Add the `*spec.container.volumeMounts*`
-to create the needed mount point. Add a new `*spec.volumes*` mentioning the configMap.
-
-====
+[source,yaml]
 ----
-$ oc edit dc router
 ...
         - name: STATS_USERNAME
           value: admin
-        - name: TEMPLATE_FILE
+        - name: TEMPLATE_FILE  <1>
           value: /var/lib/haproxy/conf/custom/haproxy-config.template
         image: openshift/origin-haproxy-routerp
 ...
         terminationMessagePath: /dev/termination-log
-        volumeMounts:
+        volumeMounts: <2>
         - mountPath: /var/lib/haproxy/conf/custom
           name: config-volume
       dnsPolicy: ClusterFirst
 ...
       terminationGracePeriodSeconds: 30
-      volumes:
+      volumes: <3>
       - configMap:
           name: customrouter
         name: config-volume
   test: false
 ...
 ----
+<1> In the `*spec.container.env*` field, add the `TEMPLATE_FILE` environment
+variable to point to the mounted *_haproxy-config.template_* file.
+<2> Add the `*spec.container.volumeMounts*` field to create the mount point.
+<3> Add a new `*spec.volumes*` field to mention the configMap.
 ====
++
 Save the changes and exit the editor.  This restarts the router.
```

Comment:
This may be outside the scope of the work here, but there's an extra space between these 2 sentences. :mag: